.github/tests/upstream-authority-cert-manager/values.yaml

```diff
@@ -2,4 +2,4 @@ spire-server:
   upstreamAuthority:
     certManager:
       enabled: true
-      issuer_name: "demo-ca"
```

Comment:
Can we keep this as well? In my case I want to reuse my existing CA. As I also use it for other usecases then Spire.

Reply:
No... you had me remove the CA with this name. The chart installs a CA if you recall. So overwriting it to a CA that no longer exists would break any test happening here.

Reply:
I think we had miscommunication there.

https://github.com/spiffe/helm-charts/pull/342#discussion_r1227319548

We still want to be able to define one ourself.

Reply:
I don't understand these PR comments anymore. The test makes more sense to verify this PR already works... not to test a custom issuer written for a test that you asked to delete

Reply:
I was the one requesting the tests be modified. @marcofranssen I'm ok with the test being switched over to test that the internally built ca is valid. The external one should still work if createCA: false

Reply:
> @marcofranssen the CA is configured in values.yaml. You can still provide one yourself. 

My fault, only realize now this was the test values.yaml 🤦‍♂️ 

> If a cert manager CR isn't going to be included in the chart, you should remove any mentioning of certManager in values.yaml. It is misleading.

It is there as an upstreamAuthority. Same like SQL support we don't install the postgres or mysql charts. We leave that up to the consumer. 

> I'm happy to add useful functions as @kfox1111 has found several and I'm pushing one more of those. I'm not going to make anymore changes just for preferences of each individual.

We are not talking on each individual. We as maintainers require 2 approvals. It is to both our interest that we take changes seriously to ensure the chart remains working for the entire community and no for single usecases. Our reviews a extensive to help the entire community.

I agree on the fact comments in PRs are way to chatty. Would be better to connect in a different manner to get things sorted.